src/utils/buildExtractStylesLoader.js

```diff
@@ -35,5 +35,9 @@ Error: ${error}
       .map(loader => `${loader}!`)
       .join('')
   );
-  return ExtractTextPlugin.extract({ fallbackLoader, loader: restLoaders });
+  return [
+    `${ExtractTextPlugin.loader().loader}?{"omit":1,"remove":true}`,
+    fallbackLoader,
+    restLoaders,
+  ].join('!');
```

Comment:
I'm fairly confident that this does not apply to webpack 1.  The latest commit for the extract-text-webpack-plugin supporting webpack 1 was back in June 15, 2016: https://github.com/webpack-contrib/extract-text-webpack-plugin/tree/webpack-1